page-changelog.php

```diff
@@ -13,7 +13,117 @@
+<li>Core: Fix issue with "(2)" appearing on shares when querying avatar with wrong casing - <a href="https://github.com/owncloud/core/issues/26271">core/#26271</a></li>
+<li>Core: Enabling an app now also analyzes dependencies at this time instead of only at install - <a href="https://github.com/owncloud/core/issues/26295">core/#26295</a></li>
+<li>Core: Reuse cached app info to avoid high load on some environments - <a href="https://github.com/owncloud/core/issues/25603">core/#25603</a></li>
+<li>Core: Show warning instead of exception when trying to run ownCloud on Windows - <a href="https://github.com/owncloud/core/issues/26208">core/#26208</a></li>
+<li>DAV: Improve chunk assembly performance for new DAV endpoint - <a href="https://github.com/owncloud/core/issues/26062">core/#26062</a></li>
+<li>DAV: New chunking now returns Etag and OC-Etag on the final MOVE - <a href="https://github.com/owncloud/core/issues/25682">core/#25682</a></li>
+<li>Files: Allow uploading empty files in the web UI - <a href="https://github.com/owncloud/core/issues/19116">core/#19116</a></li>
+<li>Files: Properly translate file summary in lists - <a href="https://github.com/owncloud/core/issues/26221">core/#26221</a></li>
+<li>Sharing: Let the share owner increase permissions - <a href="https://github.com/owncloud/core/issues/25542">core/#25542</a></li>
+<li>Federation: Fix sharing with remote user names containing spaces - <a href="https://github.com/owncloud/core/issues/25955">core/#25955</a></li>
+<li>Federation: Save some memory in sync job by releasing mount info after each user - <a href="https://github.com/owncloud/core/issues/26204">core/#26204</a></li>
+<li>Federation: Fix federated address book syncing by using the correct background job name - <a href="https://github.com/owncloud/core/issues/26202">core/#26202</a></li>
+<li>CalDAV: Add Schedule and IMip plugins when receiving webdav v1 api calendar calls - <a href="https://github.com/owncloud/core/issues/23600">core/#23600</a></li>
+<li>CardDAV: Unset photo before setting a new one - <a href="https://github.com/owncloud/core/issues/26242">core/#26242</a></li>
+<li>CardDAV: Fix for birthday entries - <a href="https://github.com/owncloud/core/issues/25636">core/#25636</a></li>
+<li>Updater: Fix web UI update in some environments - <a href="https://github.com/owncloud/updater/issues/378">updater/#378</a></li>
+<li>User_LDAP: Added OCC command to update group mappings - user_<a href="https://github.com/owncloud/ldap/issues/14">user_ldap/#14</a></li>
+<li>User_LDAP: Fix issue with "(2)" appearing on shares when refreshing users in some scenarios - <a href="https://github.com/owncloud/core/issues/25718">core/#25718</a></li>
+<li>User_external: Double verify the SMB response - <a href="https://github.com/owncloud/apps/issues/2198">apps/#2198</a></li>
+<li>Firstrunwizard: Only display it wizard when in files app - <a href="https://github.com/owncloud/firstrunwizard/issues/52">firstrunwizard/#52</a></li>
```

Comment:
Either it's a very powerful wizard that can solve all your IT problems.

Or, more likely, a typo :wink: